packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -2192,35 +2247,56 @@ class AnnotatedRegionLayer<T> extends ContainerLayer {
   /// If not provided, offset defaults to [Offset.zero].
   ///
   /// Ignored if [size] is not set.
+  ///
+  /// Unlike [OffsetLayer], this parameter [offset] does not affect the layer's
+  /// children.
```

Comment:
You might explain what it does affect, rather than just what it doesn't affect.

Reply:
The first line of the doc already explains "The [offset] is optionally used to translate the clip region for the hit-testing of [find] by [offset]." I added this line because the fact that this API resembles `OffsetLayer.offset` is a little misleading.